src/main/kotlin/com/neva/gradle/fork/config/Config.kt

```diff
@@ -102,6 +92,20 @@ abstract class Config(private val fork: ForkExtension, val name: String) {
     return templateEngine.render(template, promptedProperties)
   }
 
+  private fun propsDefine(): List<Property> {
+    val others = mutableMapOf<String, Property>()
+    val context = PropertyContext(others)
+
+    prompts.forEach { name, prompt ->
+      val definition = fork.propertyDefinitions.get(prompt.name) ?: PropertyDefinition(prompt.name)
+      val property = Property(definition, prompt, context)
+
+      others[name] = property
+    }
+
+    return others.values.sortedBy { p -> fork.propertyDefinitions.indexOf(p.name) }.toList()
```

Comment:
now order is deterministic (reflected order in `properties { define { } }` block

Reply:
like